src/App.js

```diff
@@ -4,7 +4,7 @@ import AddTodoForm from './AddTodoForm';
 
 function useSemiPersistentState () {
   const [todoList, setTodoList] = React.useState(JSON.parse(localStorage.getItem('savedTodoList')));
```

Comment:
can import useState and useEffect from react since they are named exports:
import React, { useState, useEffect } from 'react';